src/mmap_unix.rs

```diff
@@ -185,6 +190,8 @@ impl MmapRegion {
         self.flags
     }
 
+    /// Checks whether this region and `other` are backed by overlapping `FileOffset` objects.
+    ///
     /// Returns true if `self` and `other` map the same file descriptor, and the `(offset, size)`
```

Comment:
We probably don't need this phrase any more as it is clear from the previous one what is the return.

Reply:
Removed.